<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MOKELaserSim</class>
 <widget class="QMainWindow" name="MOKELaserSim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MOKELaserSim</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>408</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Number of rays to fire</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="rayCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Q</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="n0_r_input">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>1.580000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="n0_i_input">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>3.580000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Refractive Index</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="Q_r_input">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="value">
          <double>0.017700000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="Q_i_input">
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-99.000000000000000</double>
         </property>
         <property name="value">
          <double>-0.006300000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="RunSimButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Run Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopSimButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Simulation</string>
    </property>
   </widget>
   <widget class="OGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>521</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMOKE_Laser_Sim">
    <property name="title">
     <string>MOKE Laser Sim</string>
    </property>
   </widget>
   <addaction name="menuMOKE_Laser_Sim"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>oglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
